install/0000_30_machine-api-operator_07_machinehealthcheck.crd.yaml

```diff
@@ -104,7 +104,7 @@ spec:
                       type: string
                     timeout:
                       description: Expects an unsigned duration string of decimal numbers each with optional fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
Suggest to extend this description and describe "-1" behaviour there.

Reply:
I'd rather keep this undocumented for now, let's test it out with just our MHC and then negotiate with upstream to get feature parity, once we have done that, we can document it